migrations/1576059629714-remove invalid files from collection.js

```diff
@@ -52,11 +51,14 @@ module.exports = {
 		}
 
 		// find files with undefined storageFileName and not directory
-		const undefinedStorageFileNameNotDirectory = await File
-			.find({ storageFileName: 'undefined', isDirectory: false }).exec();
+		const undefinedStorageFileNameNotDirectory = await File.find({
+			storageFileName: 'undefined',
+			isDirectory: false,
+		}).exec();
 		if (undefinedStorageFileNameNotDirectory) {
-			info(`found ${undefinedStorageFileNameNotDirectory.length}`
-				+ ' files with undefinedStorageFileNameNotDirectory...');
+			info(
+				`found ${undefinedStorageFileNameNotDirectory.length}` + ' files with undefinedStorageFileNameNotDirectory...'
```

Comment:
Codacy found an issue: [Unexpected string concatenation of literals.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5989783)